spec/models/external_post_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
-describe ExternalPost do 
+describe ExternalPost do
   it { is_expected.to validate_uniqueness_of(:external_id) }
-end
\ No newline at end of file
+end
+
+describe '#username' do
+ subject { FactoryGirl.build(:external_post) }
+
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.